golang/cosmos/app/app.go

```diff
@@ -794,7 +794,21 @@ func upgrade10Handler(app *GaiaApp, targetUpgrade string) func(sdk.Context, upgr
 		app.VstorageKeeper.MigrateNoDataPlaceholders(ctx) // upgrade-10 only
 		normalizeProvisionAccount(ctx, app.AccountKeeper)
 
-		return app.mm.RunMigrations(ctx, app.configurator, fromVm)
+		mvm, err := app.mm.RunMigrations(ctx, app.configurator, fromVm)
+
+		// Just run the SwingSet kernel to finish bootstrap and get ready to open for
+		// business.
+		stdlog.Println("Rebooting SwingSet")
+		err2 := swingset.BootSwingset(ctx, app.SwingSetKeeper)
+
+		// fmt.Fprintf(os.Stderr, "BOOTSTRAP_BLOCK Returned from swingset: %s, %v\n", out, err)
+		if err2 != nil {
+			// NOTE: A failed BOOTSTRAP_BLOCK means that the SwingSet state is inconsistent.
+			// Panic here, in the hopes that a replay from scratch will fix the problem.
+			panic(err)
```

Comment:
Should be `err2`, but if you return on error right after `RunMigrations()` above, you can just reuse `err` and this line will be correct.